contents/handbook/growth/sales/new-sales.md

```diff
@@ -0,0 +1,141 @@
+---
+title: New sales
+sidebar: Handbook
+showTitle: true
+---
+
+This is our playbook for managing new sales. These are customers who have contacted sales proactively - we don't do outbound sales. Some large customers have a very prescriptive internal process that you will need to follow, but for the vast majority of our customers you will be the one driving process (and even for customers with a defined process, don't just sit back and be a passenger!).
+
+## Sales process
```

Comment:
Do our current deal steps in HubSpot match these steps one-to-one? If not maybe we can update them here or in hubspot to make tracking easier.

Reply:
They do match except for the security review, but we just do that as part of product eval in HubSpot as it usually happens in parallel and not all companies do it, so I think we're good here!